src/main/java/edu/cornell/kfs/concur/rest/xmlObjects/ExpenseDetailedReportDTO.java

```diff
@@ -0,0 +1,5 @@
+package edu.cornell.kfs.concur.rest.xmlObjects;
+
+public class ExpenseDetailedReportDTO {
```

Comment:
I was still working on figuring out what the fields will be. This will be implemented in the next Sprint. It seemed better to have retrieveExpenseDetailedReportFromConcur signature return a ExpenseDetailedReportDTO even if it is just empty for now.